src/bucket/BucketIndexImpl.cpp

```diff
@@ -84,7 +84,7 @@ BucketIndexImpl<IndexT>::BucketIndexImpl(BucketManager& bm,
         // the page size ahead of time
         if constexpr (std::is_same<IndexT, RangeIndex>::value)
         {
-            auto fileSize = fs::size(filename);
+            auto fileSize = fs::size(filename.string());
```

Comment:
Should this use `std::filesystem::file_size` instead?

Reply:
Updated.